exercises/simple-cipher/simple-cipher.spec.js

```diff
@@ -41,43 +41,43 @@ describe('Random key cipher', () => {
 describe('Incorrect key cipher', () => {
   xtest('throws an error with an all caps key', () => {
     expect(() => {
-      new Cipher('ABCDEF');
+      new Cipher('ABCDEF'); // eslint-disable-line no-new
     }).toThrow(new Error('Bad key'));
   });
 
   xtest('throws an error with a mixed-case key', () => {
     expect(() => {
-      new Cipher('ABcdEF');
+      new Cipher('ABcdEF'); // eslint-disable-line no-new
     }).toThrow(new Error('Bad key'));
   });
 
   xtest('throws an error with a numeric key', () => {
     expect(() => {
-      new Cipher('12345');
+      new Cipher('12345'); // eslint-disable-line no-new
     }).toThrow(new Error('Bad key'));
   });
 
   xtest('throws an error with an empty key', () => {
     expect(() => {
-      new Cipher('');
+      new Cipher(''); // eslint-disable-line no-new
     }).toThrow(new Error('Bad key'));
   });
 
   xtest('throws an error with a leading space', () => {
     expect(() => {
-      new Cipher(' leadingspace');
+      new Cipher(' leadingspace'); // eslint-disable-line no-new
     }).toThrow(new Error('Bad key'));
   });
 
   xtest('throws an error with a punctuation mark', () => {
     expect(() => {
-      new Cipher('hyphened-word');
+      new Cipher('hyphened-word'); // eslint-disable-line no-new
     }).toThrow(new Error('Bad key'));
   });
 
   xtest('throws an error with a single capital letter', () => {
     expect(() => {
-      new Cipher('leonardoDavinci');
+      new Cipher('leonardoDavinci'); // eslint-disable-line no-new
```

Comment:
I think this can be disabled for file rather than per-line - https://eslint.org/docs/user-guide/configuring.html#using-configuration-comments

Reply:
The `disable` here can be removed

Reply:
Yep, I just noticed I missed the last one.  Fixed in 195c7fb